resources/language/resource.language.fr_fr/strings.po

```diff
@@ -606,4 +606,4 @@ msgstr "Découvrir le contenu récement ajouté."
 
 msgctxt "#30147"
 msgid "Limits list results to about (higher numbers can cause timeouts)"
-msgstr "Limiter la liste de résultat à (un nombre élevé peut causer des erreurs)"
+msgstr "Limiter la liste de résultat à (un nombre élevé peut causer des erreurs de temps mort)"
```

Comment:
"de temps mort"? vraiment?
may I suggest:
"Limiter la liste de résultat à (un nombre élevé peut causer des erreurs liées à l'expiration de délais d'attente)"
even though I much prefer the original short one.

Reply:
I've seen some texts that use the word "temporisations" to translate "timeouts" What do you think?

Reply:
nah, I don't think "temporisations" work in this context. I really prefer the original short one, I don't think you need to be specific about the type of error. Otherwise, the translation that would be accurate is imo too long.
@pushezmadame: any other idea?

Reply:
it's okay then, keep the original short one, but you have to make a single commit not two, when it's ready i'll make the merge

Reply:
Hi, sorry for the delay.
Same opinion for the "temporatisation" choice, it means more like a queuing error.
I tried the full translation of @lekma, the auto-scroll takes to much time.

I think, we can keep the english word. I didn't check but i have already did this here : 
```
msgctxt "#30139"
msgid "Enable IPC over HTTP (use when experiencing AddonSignals timeouts)"
msgstr "Activer IPC sur HTTP (à utiliser si vous rencontrez des erreurs AddonSignals timeouts)"
```

Ok for everybody ?
> "Limiter la liste de résultat à (un nombre élevé peut causer des erreurs de timeouts"

Reply:
it's okay for me

Reply:
yep, fine